### PR TITLE
infoschema: add tidb_hot_regions_history virtual table

### PR DESCRIPTION
**Pull request by IcePigZDB**
Signed-off-by: IcePigZDB <icepigzdb@gmail.com>

Problem Summary: Create a new table `TIDB_HOT_REGIONS_HISTORY` in the `INFORMATION_SCHEMAT` database to retrieve history hotspot information stored by PD periodically.
### What problem does this PR solve?
Proposal: [pingcap/tidb#27487](https://github.com/pingcap/tidb/pull/27487)
Associated Issue: #25281

Associated PRs in TiDB:
PRs splited from oversize PR #27224 in merge order :
1. #27373 **<- current one** 
2. #27374
3. #27375
4. #27899
5. #27900   

Associated PR in PD:  
 PRs splited from old oversize one: tikv/pd/pull/3989 
* tikv/pd/pull/4019
* tikv/pd/pull/4020
* tikv/pd/pull/4021
### Check List 

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
select * from information_schema.tidb_hot_regions_history where update_time>='2019-10-10 10:10:10' and update_time<'2019-10-11 10:10:10'
```

### CODE REVIEW

File: infoschema/tables.go [hunk 1]
```diff
@@ -879,6 +882,24 @@ var TableTiDBHotRegionsCols = []columnInfo{
 	{name: "FLOW_BYTES", tp: mysql.TypeLonglong, size: 21},
 }
 
+// TableTiDBHotRegionsHistoryCols is TiDB hot region history mem table columns.
+var TableTiDBHotRegionsHistoryCols = []columnInfo{
+	{name: "UPDATE_TIME", tp: mysql.TypeVarchar, size: 64},
```
Comment (nolouch) [inline infoschema/tables.go:887]: use timestamp?

File: infoschema/tables.go [hunk 2]
```diff
@@ -879,6 +882,24 @@ var TableTiDBHotRegionsCols = []columnInfo{
...
+	{name: "DB_NAME", tp: mysql.TypeVarchar, size: 64},
+	{name: "TABLE_NAME", tp: mysql.TypeVarchar, size: 64},
+	{name: "TABLE_ID", tp: mysql.TypeLonglong, size: 21},
+	{name: "INDEX_NAME", tp: mysql.TypeVarchar, size: 64},
+	{name: "INDEX_ID", tp: mysql.TypeLonglong, size: 21},
+	{name: "REGION_ID", tp: mysql.TypeLonglong, size: 21},
+	{name: "STORE_ID", tp: mysql.TypeLonglong, size: 21},
+	{name: "PEER_ID", tp: mysql.TypeLonglong, size: 21},
```
Comment (nolouch) [inline infoschema/tables.go:895]: may need to know whether it is a leader.

**Review by nolouch — APPROVED** [review]
lgtm

**Review by xhebox — APPROVED** [review]
I guess the test will be added in later PRs? Will merge if that is the case.

**Comment by IcePigZDB** [comment]
> I guess the test will be added in later PRs? Will merge if that is the case.

Move test into two new PRs.

**Pull request merged**